### PR TITLE
Juan, Eduardo, Manuel and Jaime added and filled file "roles.md"

### LINKED ISSUE #47: Create the roles.md file
Create a folder named `unit1`, and a file `roles.md`
In the `database-concepts.md` file, explain the following concepts:

DB-related roles:

DBA
data analyst
data engineer
data scientist
data architect

### PR DESCRIPTION
**Pull request by Karls3fni**
Closes #47

### CODE REVIEW

**Review by bananas24 — CHANGES_REQUESTED** [review]
@JaimeRuiz06 remove the underscores from the entries, and this should be ready to go

**Comment by tamasi17** [comment]
Looks good 🦕

File: unit1/roles.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Data Base Roles
+
+1_ Data Engineer
```
Comment (bananas24) [inline unit1/roles.md:3]: Remove the underscores from the titles

**Review by AshysCoffee — APPROVED** [review]

**Review by YuUs01 — APPROVED** [review]
ok

**Review by Marcos-18-11 — APPROVED** [review]

**Review by symn369 — APPROVED** [review]

**Review by bananas24 — APPROVED** [review]

**Review by Ces216 — APPROVED** [review]

**Review by tamasi17 — APPROVED** [review]
Good job

**Pull request merged**